src/sidebar/store/test/util-test.js

```diff
@@ -134,5 +140,19 @@ describe('reducer utils', function () {
       assert.equal(bound.countAnnotations1(), 1);
       assert.equal(bound.countAnnotations2(), 1);
     });
+
+    it('binds root selectors to current value of root state', () => {
+      const getState = sinon.stub().returns({
+        namespace1: {
+          annotations: [{ id: 1 }],
+        },
+        namespace2: {
+          annotations: [{ id: 2 }],
+        },
+      });
+      const bound = util.bindSelectors(fixtures.selectors, getState);
+      assert.equal(bound.rootCountAnnotations1(), 1);
```

Comment:
Would it be easier to spot problems if the two selectors returned different results?